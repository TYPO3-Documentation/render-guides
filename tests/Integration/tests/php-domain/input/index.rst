=====================================
PHP Interface with implicit namespace
=====================================

..  php:interface:: TYPO3\CMS\Core\TestInterface

    Lorem Ipsum Dolor!

    ..  php:method:: setDate(int $year, int $month, int $day)

            Set the date.

    ..  php:method:: getDate(): int

            Get the date.

See also :php:interface:`TYPO3\CMS\Core\TestInterface`,
for short :php:interface:`TestInterface <TYPO3\CMS\Core\TestInterface>`!
