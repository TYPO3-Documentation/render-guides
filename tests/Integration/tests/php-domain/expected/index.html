<!-- content start -->
    <div class="section" id="php-interface-with-implicit-namespace">
    <h1>PHP Interface with implicit namespace</h1>

            <dl class="php interface">
    <dt class="sig sig-object php"
        id="typo3-cms-core-testinterface">
        <em class="property"><span class="pre">interface</span> </em>
            <span class="sig-prename descclassname"><span class="pre">\TYPO3\CMS\Core\</span></span>
<span class="sig-name descname"><span class="pre">TestInterface</span></span>

    </dt>
    <dd>
        <p>Lorem Ipsum Dolor!</p><dl class="php method">
    <dt class="sig sig-object php" id="setdate">
        
<span class="sig-name descname"><span class="pre">setDate</span></span>
<span class="sig-paren">(</span>
<em class="sig-param"><span class="pre">int $year</span></em>, <em class="sig-param"><span class="pre">int $month</span></em>, <em class="sig-param"><span class="pre">int $day</span></em><span class="sig-paren">)</span>

    </dt>
    <dd>
        <p>Set the date.</p>
    </dd>
</dl>
<dl class="php method">
    <dt class="sig sig-object php" id="getdate">
        
<span class="sig-name descname"><span class="pre">getDate</span></span>
<span class="sig-paren">(</span>
<span class="sig-paren">)</span>
<em class="sig-returns"><span class="pre">: int</span></em>

    </dt>
    <dd>
        <p>Get the date.</p>
    </dd>
</dl>

    </dd>
</dl>

            <p>See also <a href="/index.html#typo3-cms-core-testinterface">\TYPO3\CMS\Core\TestInterface</a>
,
for short <a href="/index.html#typo3-cms-core-testinterface">TestInterface</a>
!</p>
    </div>

<!-- content end -->